primitive-types/src/lib.rs

```diff
@@ -30,6 +30,11 @@ pub enum Error {
 	Overflow,
 }
 
+construct_uint! {
+	/// 64-bit unsigned integer.
+	#[cfg_attr(feature = "scale-info", derive(TypeInfo))]
+	pub struct U64(1);
```

Comment:
I'm not keen on adding this type to primitives. 
In fact, we wanted to remove it altogether, see #473 for the context.

Reply:
My bad for not reading this issue thoroughly, and thanks for pointing it out. 

I understand now why you suggested adding this to `ethereum-types` a year ago in #473. Just for clarity, you are still okay with adding this functionality in general, right?

My motivation behind this is that there are still libraries that use U64 in certain cases (eg. [ethers-rs block-num](https://github.com/gakonst/ethers-rs/blob/master/ethers-core/src/types/block.rs#L42)) and conversion to other uint types is currently not the most elegant as already pointed out in the referenced issue.